src/main.cpp

```diff
@@ -58,10 +58,16 @@ void displayGUI() {
 
         int deviceType = sensors.getPmDeviceTypeSelected();
         uint16_t mainValue = 0;
-        if ((deviceType > 0 && deviceType <= 2) || deviceType == sensors.SDS011)
```

Comment:
we need validate the PM2.5 sensors, whay is the reason for remove SDS011 validation?

Reply:
Because all the sensor <= 3 are PM2.5 included the SDS011

Reply:
ok, you changed the order. Well, we need maybe replanning it, we could have issues in the future. For now it's right.